core/src/com/unciv/ui/overviewscreen/NotificationsOverviewTable.kt

```diff
@@ -64,7 +64,10 @@ class NotificationsOverviewTable(
             val label = WrappableLabel(notification.text, labelWidth, Color.BLACK, 20)
 
             notificationTable.add(label)
-            notificationTable.background = ImageGetter.getRoundedEdgeRectangle()
+            notificationTable.background =  if (UncivGame.Current.settings.coloredNotifications)
+                                                ImageGetter.getRoundedEdgeRectangle(notification.color.color)
+                                            else
+                                                ImageGetter.getRoundedEdgeRectangle()
```

Comment:
@Azzurite It is a setting. The only problem is it doesn't update the notif scroll until "next turn" is pressed